tests/compas/geometry/test_curves_polyline.py

```diff
@@ -431,6 +449,16 @@ def test_polyline_tangent_at_point(coords, input, expected):
     assert expected == Polyline(coords).tangent_at_point(input)
 
 
+@pytest.mark.parametrize("input, expected", [((0, 0, 0), 0.0), ((1, 0, 0), 0.5), ((1, 1, 0), 1.0), ((2, 0, 0), None)])
```

Comment:
i think the space behind the comma is something the ipy will not like...

Reply:
Oops that's my bad, I'm sorry